doc/act-rules-format.md

```diff
@@ -0,0 +1,35 @@
+# W3C Standardized Rules
+
+Deque Systems is one of leading organizations in the development of standardized accessibility conformance testing rules. The [axe-core rules proposal format](./rule-proposal.md) is an adaptation of the [Accessibility Conformance Testing Rules Format](https://www.w3.org/TR/act-rules-format/).
+
+There are two ways a rule written in the axe-core rule format can be transformed into the ACT Rules format:
+
+## Method 1: Create a single rule
```

Comment:
Best practices for rules should be added:

1) Rules should only have one `none` check so that the error message is specific
2) Rules should not combine `any` and `none`, these should be broken out into separate rules
3) Checks should each only test a single specific case (either a passing technique or a single failing test)